docs/HyperIndex/Advanced/config-schema-reference.md

```diff
@@ -56,10 +58,10 @@ name: MyIndexer
 
 Ecosystem of the project.
 
-- **type**: `anyOf(unknown | null)`
+- **type**: `anyOf(object<EcosystemTag> | null)`
 
 Variants:
-- `1`: [EcosystemTag](#ecosystemtag)
+- `1`: [EcosystemTag](#def-ecosystemtag)
```

Comment:
There were clashes, eg on the old site there was no way to link to this part of the page (since they both had the same link: https://docs.envio.dev/docs/HyperIndex/config-schema-reference#fieldselection)

<img width="1547" height="973" alt="image" src="https://github.com/user-attachments/assets/43812ed3-5255-4224-9c62-44182baf8193" />

unavailable via link since overshadowed:
![Uploading image.png…]()